syntaxes/elixir.json

```diff
@@ -168,6 +171,18 @@
         }
       }
     },
+    {
+      "comment": "Qualified map access without parenthesis",
+      "match": "\\s*(\\.)\\s*([a-z_]\\w*[!?]?)",
```

Comment:
Huh, I'm surprised whitespace is permitted after the dot. Nice!

Thanks for catching this

Reply:
While not idiomatic, things like this are valid:

```elixir
map.
  key
```

I can't any credit for remembering this fact, though -- I just copied and modified a similar rule :)